### ISSUE TITLE
Constructing a vector from a 2d view gives incorrect result

### ISSUE THREAD

**Issue by romeric** [romeric/Fastor#47]
The following gives incorrect result

~~~c++
        Tensor<double,2,3> a;
        a.iota();
        Tensor<double,3> b = a(0,all);
~~~

whereas this one gives the correct result

~~~c++
        Tensor<double,2,3> a;
        a.iota();
        Tensor<double,1,3> b = a(0,all);
~~~

**Comment by romeric** [comment]
Related issue #46

**Comment by romeric** [comment]
dd92e18 temporarily has turned off so this issue does not happen until a real fix is found for the issue.

**Comment by romeric** [comment]
Proper fixed applied in f877d7e.
